README.md

```diff
@@ -146,7 +146,7 @@ The default environment passed along with every log sent through this instance.
 * Default: `5 * time.Second`
 * Values: `10 * time.Second`
```

Comment:
having `Values` might be confusing like you are telling them these are the values they can use - I'd suggest to remove this line + is the type `time.duration` or `int` in seconds which you later multiply by `time.Second`?

Reply:
The purpose of the value field is just to give samples to the user of what they can use for a particular field (like FlushInterval) or example. Have it for the other fields (env, hostname etc.) so thought it would be useful to have it for FlushInterval too.

Also the type of FlushInterval is time.duration: https://github.com/logdna/logdna-go/blob/master/logger/logger.go#L19

Reply:
I might change to "Example Value" instead of `Values` since they're really intended to show how to format or enter an example, like on line 187 where the value is a string, but it should be an IP address:

```suggestion
* Example Value: `10 * time.Second`
```

Reply:
I agree with @nimbinatus's suggestion

Reply:
should I change the other fields to example value as well @nimbinatus @smusali?- want to be consistent in this readme and also with readmes in our other clients. for example looking at the nodejs client:

##### level

* _**Optional**_
* Type: `String`
* Default: `Info`
* Values: `Debug`, `Trace`, `Info`, `Warn`, `Error`, `Fatal`, `YourCustomLevel`
* Max Length: `32`

The level passed along with this log line.

##### app

* _**Optional**_
* Type: `String`
* Default: `''`
* Values: `YourCustomApp`
* Max Length: `32`

The app passed along with this log line.

Reply:
I would. It's a lot clearer for future users. I think you can have multiple example values, as with the nodejs client. But I could see someone thinking those are the only values (and literally thinking the value `YourCustomLevel` is the actual value to pass).